system_meta/admin.py

```diff
@@ -0,0 +1,54 @@
+"""Django Admin for system_meta app"""
+
+from django.contrib import admin
+from safedelete.admin import SafeDeleteAdmin, SafeDeleteAdminFilter, highlight_deleted
+
+from system_meta.models import IntegratedSystem, Product
+
+
+class IntegratedSystemAdmin(SafeDeleteAdmin):
```

Comment:
Unless I'm missing something when I go into the edit page for a specific `IntegratedSystem` I'm not seeing the `is_active` field or a way to "delete" the object. So I think the only way to change that state is through the shell atm.

Reply:
The softdelete library overrides the `delete` method, so just using the normal Delete button on the edit page should work. You can undelete or hard delete things through the list page's Actions dropdown too.